src/dmd/doc.d

```diff
@@ -108,7 +106,7 @@ private class Section
     void write(Loc loc, DocComment* dc, Scope* sc, Dsymbols* a, OutBuffer* buf)
     {
         assert(a.dim);
-        if (namelen)
+        if (!name.length)
```

Comment:
You probably didn't intend to invert the condition.

Reply:
Indeed, thanks